src/server.ts

```diff
@@ -48,6 +48,8 @@ export function walkTree(
     //   https://github.com/facebook/react/blob/master/src/renderers/shared/stack/reconciler/ReactCompositeComponent.js#L66
     if (Component.prototype && Component.prototype.isReactComponent) {
       const instance = new Component(props, context);
+      instance.props = props;
+      instance.context = context;
```

Comment:
These three constructors are valid for react:
```
constructor() {
super();
}

constructor(props) {
super(props);
}

constructor(props, context) {
super(props, context);
}
```
But only the last one is covered by `new Component(props, context)`. At least thats what I suppose.